pipeline_steps/maas.groovy

```diff
@@ -63,4 +63,30 @@ cd ${env.WORKSPACE}/rpc-gating/scripts
   return token_url.trim().tokenize(" ")
 }
 
+def entity_cleanup(Map args){
+  common.conditionalStep(
+    step_name: 'Cleanup',
+    step: {
+      withCredentials(common.get_cloud_creds()) {
+        dir("rpc-gating/playbooks") {
+          common.install_ansible()
+          pyrax_cfg = common.writePyraxCfg(
+            username: env.PUBCLOUD_USERNAME,
+            api_key: env.PUBCLOUD_API_KEY
+          )
+          withEnv(["RAX_CREDS_FILE=${pyrax_cfg}"]) {
+            common.venvPlaybook(
+              playbooks: ['aio_maas_cleanup.yml'],
+              venv: ".venv",
+              vars: [
+                "server_name": args.instance_name,
+              ]
```

Comment:
Not worth blocking, but since the card for this refactor cites MaaS API instability, should this or the maas playbooks have a retry included?